bumble/gatt_client.py

```diff
@@ -558,23 +555,32 @@ async def subscribe(self, characteristic, subscriber=None):
             logger.warning('subscribing to characteristic with no CCCD descriptor')
```

Comment:
OOS, but this warning is wrong. we don't know if theres no CCCD as we just discovered descriptors and haven't checked if CCCD exists

Reply:
The line above gets the CCCD only if it exists:
`cccd = characteristic.get_descriptor(GATT_CLIENT_CHARACTERISTIC_CONFIGURATION_DESCRIPTOR)`